shared/test/integration/tbpl-manifest.json

```diff
@@ -41,6 +41,11 @@
     "apps/verticalhome/test/marionette/app_hosted_use_cached_icon_test.js": "Bug 1037194 - TEST-UNEXPECTED-FAIL | Vertical Home - Hosted app cached icon fetch fallback to default icon",
     "apps/verticalhome/test/marionette/collection_pin_result_uninstall_test.js": "Bug 1098310 - TEST-UNEXPECTED-FAIL | Vertical - Collection uninstall pinned collection web result",
     "apps/verticalhome/test/marionette/collection_rename_test.js": "Bug 1090756 - TEST-UNEXPECTED-FAIL | Vertical - Collection Rename rename collection",
-    "apps/verticalhome/test/marionette/group_collapse_drag_into_test.js": "Bug 1125696 - Intermittent failing test, TEST-UNEXPECTED-FAIL | Vertical - Group check dragging icon into collapsed group"
+    "apps/verticalhome/test/marionette/group_collapse_drag_into_test.js": "Bug 1125696 - Intermittent failing test, TEST-UNEXPECTED-FAIL | Vertical - Group check dragging icon into collapsed group",
+    "apps/system/test/marionette/rocketbar_activity_layout_test.js": "Bug 1117770 - Remove e.me",
+    "apps/verticalhome/test/marionette/search_test.js": "Bug 1117770 - Remove e.me",
+    "apps/search/test/marionette/bookmark_web_result_test.js": "Bug 1117770 - Remove e.me",
```

Comment:
I still think we should just remove the tests that we are not going to utilize in the future. If we are not going to be bookmarking web results, this would be this and the next two tests I believe.